Source/IGListKit/IGListBindingSectionControllerSelectionDelegate.h

```diff
@@ -61,6 +61,21 @@ NS_SWIFT_NAME(ListBindingSectionControllerSelectionDelegate)
 didUnhighlightItemAtIndex:(NSInteger)index
                 viewModel:(id)viewModel;
 
+/**
+ Tells the delegate that a cell has requested a menu configuration.
+
+ @param sectionController The section controller the request of a menu configuration occurred in.
+ @param index The index of the cell that is being longed tap.
+ @param point The point of the tap on the cell.
+ @param viewModel The view model that was bound to the cell.
+ 
+ @return An object that conforms to `UIContextMenuConfiguration`.
+ */
+- (UIContextMenuConfiguration * _Nullable)sectionController:(IGListBindingSectionController *)sectionController
```

Comment:
I'm also not super familiar with `IGListKit`, but this seems like an unnecessary forwarding of the call — after all it's `IGListBindingSectionController**Selection**Delegate`, not a "general delegate" — why would a "delegate that receives selection events from cells" be interested in context menu events?